conda/_vendor/tqdm/utils.py

```diff
@@ -261,7 +261,6 @@ def _screen_shape_windows(fp):  # pragma: no cover
 def _screen_shape_tput(*_):  # pragma: no cover
     """cygwin xterm (windows)"""
     try:
-        # from shlex import split as ss
         from conda.auxlib.compat import shlex_split_unicode as ss
```

Comment:
That's an interesting line given our recent work with quoting. Shouldn't we rely on that instead?

Reply:
Uuuh, this seems like a can of worms, looking at cf5506804ba7797a3fb6180c55bc5b7811226030.

Reply:
See https://github.com/conda/conda/issues/11335